portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,39 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+
+body{
+    margin: auto;
+    background-color: lightblue;
+}
+
+h1{
+    text-align: center;
+    border-style: solid;
+    border-width: medium;
+    border-color: red;
```

Comment:
FYI, our practice is to sort all the CSS attributes alphabetically. But this is just a nit (nitpicking).